.bazelrc

```diff
@@ -61,6 +61,9 @@ build --host_per_file_copt='external/zlib@-Wno-unknown-warning-option,-Wnodeprec
 build --per_file_copt=external/com_google_tcmalloc@-DANNOTATE_MEMORY_IS_INITIALIZED=ABSL_ANNOTATE_MEMORY_IS_INITIALIZED
 build --per_file_copt=external/com_google_protobuf@-Wno-deprecated-declarations,-Wno-deprecated-pragma
 build --host_per_file_copt=external/com_google_protobuf@-Wno-deprecated-declarations,-Wno-deprecated-pragma
+# Enables heap allocation debug checks for local handles.
+# Ref: https://chromium-review.googlesource.com/c/v8/v8/+/4905902
+build --per_file_copt='external/v8@-DV8_ENABLE_LOCAL_OFF_STACK_CHECK'
```

Comment:
Actually I was wrong about this only being needed when building V8: Since the define present in include/v8-local-handle.h and include/v8-fast-api-calls.h we also need to apply it when building our own code. Let's change this to a regular `copt`.